controllers/SitemapController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Pimcoresitemapplugin_SitemapController
+ *
+ * @author Yann Weyer
+ */
```

Comment:
I think there'll probably be a few quick style things like this, but, we have a common file header across all of the PHP source files in the repository that's like this:

```php
<?php

/**
 * This file is part of the pimcore-sitemap-plugin package.
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```